client/navigation/style.scss

```diff
@@ -33,6 +33,10 @@ body.is-wc-nav-expanded {
 		width: $navigation-width;
 		height: 100%;
 	}
+
+	font > .xe-notice {
```

Comment:
These classes don't quite seem to work for me with errors.  I see these classes:

`xdebug-error xe-warning`